### PR TITLE
[ci-skip]: add 'none' as a new option for transaction manager.

### LINKED ISSUE #2234: feat(quorum): add 'none' as a new option for transaction manager
**Is your feature request related to a problem? Please describe.**
Currently, in Hyperledger Quorum, privacy is an add-on feature that is not supported out of the box. The parameter `transaction_manager` is bound to have two values - "tessera" or "orion"- meaning that users must provide a value, even if they do not intend to use the privacy feature. This can result in validation errors or unnecessarily require users to provide additional fields.

**Describe the solution you'd like**
To address this issue, adding an option for "none" as a valid value for the `transaction_manager` parameter would be beneficial. This would allow users to indicate that they do not want to use the privacy feature, without encountering validation errors or having to provide additional fields.

**Describe alternatives you've considered**
Currently, users must provide a value for `transaction_manager` and its related fields like `tm_version`, `tm_tls`, `tm_trust`, and `tm_nodes` even if they do not intend to use the privacy feature.

**Suggestion** 
Introducing an option for "none" as a valid value for `transaction_manager` would give users more flexibility in configuring their network and help avoid unnecessary validation errors or additional configuration.

### PR DESCRIPTION
**Pull request by saurabhkumarkardam**
### **Commit to be reviewed**
---
feat(quorum): add 'none' as a new option for transaction manager.

```
Updated the following files to introduce the new option for transaction manager:
  1. deployment.yaml file of chart node_quorum_member
  2. memberquorum.tpl template
  3. deploy-network.yaml file
  4. network-schema.json
  5. sample network-quorum.yaml file
  6. sample network-quorum-constellation.yaml
```

fixes #2234

### CODE REVIEW

File: platforms/quorum/configuration/deploy-network.yaml [hunk 1]
```diff
@@ -163,6 +163,10 @@
     loop: "{{ network['organizations'] }}"
     when: network.config.transaction_manager == 'constellation'
 
+  - name: Print transaction manager configuration
```
Comment (sownak) [inline platforms/quorum/configuration/deploy-network.yaml:166]: please remove debug statement

File: platforms/quorum/charts/node_quorum_member/templates/deployment.yaml [hunk 2]
```diff
@@ -215,6 +220,15 @@ spec:
             args=" ${args} --unlock 0 "
           fi
 
+          ptm_url="{{ .Values.tessera.clienturl | quote }}"
+          if [ -n "$ptm_url" ]
```
Comment (sownak) [inline platforms/quorum/charts/node_quorum_member/templates/deployment.yaml:224]: Please use if [ {{ $.Values.tm.type }} != "none" ] condition here as well

**Review by sownak — CHANGES_REQUESTED** [review]

**Review by jagpreetsinghsasan — APPROVED** [review]
Looks good to me (previous review comments seem to be incorporated)

File: platforms/quorum/charts/crypto_raft/templates/job.yaml [hunk 3]
```diff
@@ -15,6 +15,7 @@ metadata:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
+    app: "{{ $.Values.metadata.name }}"
```
Comment (suvajit-sarkar) [inline platforms/quorum/charts/crypto_raft/templates/job.yaml:18]: remove duplicate app labels

**Review by sownak — APPROVED** [review]

**Pull request merged**